.gitignore

```diff
@@ -7,7 +7,7 @@
 .svn
 *~
 manifest.appcache
-
+/.idea
```

Comment:
That's good to have! I also use Jetbrains' product and I've never took the time to include the rule. Nonetheless, as we don't own this, this might require to be done in another bug.

Out of curiosity, what's the difference between `.idea` and `/.idea`?